magento/magento2
# fix for Character Encoding - Full Name - Welcome, Te&#039;st! #36819

### Description (*)
  because data-bind was used as text so in this pull request i have used bindHtml.
  currently its looking good i have also tried to set JS code in there as well like this.
  data-bind="bindHtml: new String('<?= $escaper->escapeHtml(__('Welcome , %1! ', '%1')) ?>').replace('%1', 'alert(11)')">
  
  And it is being escaped well.


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36819

### Manual testing scenarios (*)

1.  create or edit customers name with waving character ' in the name, 
2.  it should displayed well with '  character in header welcome message

### Questions or comments


### Contribution checklist (*)
